flink-formats/flink-json/src/main/java/org/apache/flink/formats/json/debezium/DebeziumJsonDecodingFormat.java

```diff
@@ -150,13 +150,16 @@ public Object convert(GenericRowData row, int pos) {
 
 		INGESTION_TIMESTAMP(
 			"ingestion-timestamp",
-			DataTypes.TIMESTAMP_WITH_LOCAL_TIME_ZONE(3).notNull(),
+			DataTypes.TIMESTAMP_WITH_LOCAL_TIME_ZONE(3).nullable(),
```

Comment:
Is this supposed to be part of this PR?

Reply:
I spotted this while going through the docs of Debezium and corrected it inline.